utils/webhook_cache.py

```diff
@@ -17,31 +17,31 @@ def store_webhooks(self, webhooks: list, identifiers: list, servers: list):
             webhook = webhooks[index]
             identifier = identifiers[index]
             server = servers[index]
-            if not server in self.__webhooks.keys():
-                self.__webhooks.update({server: {identifier: webhook}})
-            self.__webhooks[server].update({identifier: webhook})
+            self.__webhooks.setdefault(server, {})[identifier] = webhook
         return len(self.__webhooks)
 
-    def get_webhooks(self, server: int or str):
+    def get_webhooks(self, server: Union[int, str]):
```

Comment:
```suggestion
    def get_webhooks(self, server: int | str):
```
This isn't really required, but it might be nice to use this as an opportunity to switch to pipe operators from typing.Union.

For bridge.py I'd leave it as is, since I'll be rewriting it in the near future anyways, so I can do that then.